meta/runtime.yml

```diff
@@ -588,3 +593,5 @@ plugin_routing:
       redirect: community.docker.docker_machine
     docker_swarm:
       redirect: community.docker.docker_swarm
+    kubevirt:
+      redirect: community.kubevirt.kubevirt
```

Comment:
There are redirects missing for plugins/module_utils/kubevirt.py, plugins/doc_fragments/kubevirt_common_options.py and plugins/doc_fragments/kubevirt_vm_options.py

Reply:
Good catch.

Reply:
@felixfontein fixed by https://github.com/ansible-collections/community.general/pull/1317/commits/95c858fe45ec13f2b08c4210ff856942935f80ae